### ISSUE TITLE
Mismatch with new DNAScope ML model format

### ISSUE THREAD

**Issue by Jakob37** [nf-core/raredisease#568]
### Description of the bug

The Sentieon DNAScope takes a flag `--model` which is expected to point to a file with a pre-trained model.

In their latest updates, the models are provided in `.bundle` format. Here the model is accessed as such:

```
--model /path/to/models.bundle/dnascope.model
```

(More info in issue here: https://github.com/Sentieon/sentieon-models/issues/4)

But, the file is actually named `models.bundle`.

So if you point to the `models.bundle` directly, the Sentieon process does currently not find the file. If pointing directly to `dnascope.model`, Nextflow will detect it as missing and will not run.

My temporary solution is to update the Sentieon DNA-scope module code as such:

```
def model_cmd                 = ml_model                           ? " --model ${ml_model}/dnascope.model"   : ''
```

**Comment by asp8200** [comment]
Yeah, we need to fix that. In the mean time, you can unpack the bundle using `ar` and get hold of the module-file you need.

**Comment by Jakob37** [comment]
> Yeah, we need to fix that. In the mean time, you can unpack the bundle using `ar` and get hold of the module-file you need.
> 
> EDIT: Your fix is also fine.

Didn't know that, thanks!

**Comment by maxulysse** [comment]
xref: https://github.com/nf-core/sarek/issues/1399

**Comment by ramprasadn** [comment]
Fixed in #912
